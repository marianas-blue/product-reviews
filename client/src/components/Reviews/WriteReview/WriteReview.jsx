/* eslint-disable no-useless-constructor */
/* eslint-disable react/prefer-stateless-function */
import React from 'react';
import { Text, Button } from './WriteReview.style';

class WriteReview extends React.Component {
  render() {
    return (
      <div>
        <Text>
          See all 453 reviews
        </Text>

        <Button>
          Write a customer review
        </Button>
      </div>
    );
  }
}

export default WriteReview;
