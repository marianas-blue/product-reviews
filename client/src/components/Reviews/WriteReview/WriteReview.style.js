import styled from 'styled-components';

export const Text = styled.div`
  color: #0066c0;
  font-size: 13px;
  font-weight: bold;
  :hover {
    text-decoration: underline;
    color: #c60;
  }
`;

export const Button = styled.button`
  background: #eff1f3;
  border-radius: 3px;
  border-color: #ADB1B8 #A2A6AC #8D9096;
  border-style: solid;
  border-width: 1px;
  cursor: pointer;
  display: inline-block;
  margin: 8px 0px 8px 0px;
  width: 160px;
  height: 29px;
  font-family: Lato, sans-serif;
`;