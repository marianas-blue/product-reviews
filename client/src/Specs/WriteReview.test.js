/* eslint-disable react/jsx-filename-extension */
/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import WriteReview from '../components/Reviews/WriteReview/WriteReview';

describe('WriteReview', () => {
  it('should render correctly', () => {
    const component = shallow(<WriteReview />);

    expect(component).toMatchSnapshot();
  });
});
