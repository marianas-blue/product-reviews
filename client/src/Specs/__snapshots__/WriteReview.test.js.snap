// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WriteReview should render correctly 1`] = `
<div>
  <styled.div>
    See all 453 reviews
  </styled.div>
  <styled.button>
    Write a customer review
  </styled.button>
</div>
`;
